src/modules/account/selectors/core-stats.js

```diff
@@ -0,0 +1,103 @@
+import { createSelector } from 'reselect';
+import store from 'src/store';
+import { selectAccountTradesState, selectBlockchainState, selectOutcomesDataState } from 'src/select-state';
+import { augur, abi } from 'services/augurjs';
+import { dateToBlock } from 'utils/date-to-block-to-date';
+import { formatEther } from 'utils/format-number';
+import { ZERO } from 'modules/trade/constants/numbers';
+import { selectLoginAccount } from 'modules/account/selectors/login-account';
+import selectLoginAccountPositions from 'modules/my-positions/selectors/login-account-positions';
+import getValue from 'utils/get-value';
+
+export default function () {
+  return selectCoreStats(store.getState());
+}
+
+export const selectOutcomeLastPrice = (marketOutcomeData, outcomeID) => {
+  if (!marketOutcomeData || !outcomeID) return null;
+  return (marketOutcomeData[outcomeID] || {}).price;
+};
+
+// Period is in days
+export const createPeriodPLSelector = period => createSelector(
+  selectAccountTradesState,
```

Comment:
Per the docs, shouldn’t there be just two args passed to the `createSelector`: `an array of input-selectors and a transform function as its arguments`?